src/Notifier.js

```diff
@@ -16,8 +16,10 @@ limitations under the License.
 
```

Comment:
The plan was to move it over to be a Module although this PR removes the Module system. I'm somewhat concerned that we're dealing with tradeoff's we've made in the name of making a flexible, re-usable SDK by removing all the flexibility and worried that we're going to end up in the same situation again when we add the flexibility back in again.